docs/source/usage/workflows/numberOfCells.rst

```diff
@@ -8,10 +8,14 @@ Setting the Number of Cells
 Together with the grid resolution in :ref:`grid.param <usage-params-core>`, the number of cells in our :ref:`.cfg files <usage-tbg>` determine the overall size of a simulation (box).
 The following rules need to be applied when setting the number of cells:
 
-Each GPU needs to:
+Each device needs to:
 
 #. contain an integer *multiple* of supercells
 #. at least *three* supercells
+#. for non periodic boundaries the absorber must fit into the local volume
+
+The grid size will be automatically adjusted if the conditions above are not fulfilled. 
+This behavior can be disabled by using the command line option `--autoAdjustGrid off`
```

Comment:
needs two `` in `r.st` :)

Reply:
:-( ok will fix it

Reply:
yeah, it's super confusing that rst and md have this subtle difference xD